homeassistant/components/smartthings/sensor.py

```diff
@@ -526,3 +540,58 @@ def state(self):
             return three_axis[self._index]
         except (TypeError, IndexError):
             return None
+
+
+class SmartThingsPowerConsumptionSensor(SmartThingsEntity, SensorEntity):
```

Comment:
This implementation is only exposing `energy` when the capability has 4 other useful data values. Perhaps add a constructor parameter to indicate which sub-attribute it's representing, like ThreeAxis does.